app/component/map/popups/LocationPopup.js

```diff
@@ -121,6 +121,21 @@ class LocationPopup extends React.Component {
     const [address, place] = splitStringToAddressAndPlace(
       this.state.location.address,
     );
+
+    const { lat, lon } = this.props;
+
+    const reporterUrl =
+      (/^(localhost:8080|herrenberg.staging.stadtnavi.eu)$/.test(
+        window.location.host,
+      ) &&
+        `https://maengelmelder.service-bw.de/?lat=${lat}&lng=${lon}`) ||
+      null;
+    // In DT1 has this additional condition, DT2 doesn't use postal code.
```

Comment:
Could you recheck if the postal code is gone in DT2?

When I execute the following URL https://photon.stadtnavi.eu/pelias/v1/reverse?point.lat=48.59670578069282&point.lon=8.867340087890627&boundary.circle.radius=0.1&lang=de&size=1&layers=address&zones=1 I can see it.